src/docs/release-notes/R17/Summary.mdx

```diff
@@ -0,0 +1 @@
+Updated content for the design pattern pages, added more components to the PX Blue stickersheet, new Angular components for [Dropdown Toolbar](https://pxblue-components.github.io/angular/?path=/info/components-dropdown-toolbar--readme) and [Navigation Rail](https://pxblue-components.github.io/angular/?path=/story/components-drawer--with-a-nav-rail), new React component for [Dropdown Toolbar](https://pxblue-components.github.io/react/?path=/info/components-dropdown-toolbar--get-read-me-story), new React Native components for [Spacer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--spacer) and [List Item Tag](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--list-item-tag), built a re-usable authentication workflow for React.
```

Comment:
add a link to patterns page here too

Reply:
Added a link out to the design patterns page.